client/src/components/atoms/Modal/Modal.tsx

```diff
@@ -27,7 +27,7 @@ const Modal: React.FC<ModalProps> = ({
             tabIndex={-1}
             role="dialog"
           >
-            <code>{content}</code>
+            <div dangerouslySetInnerHTML={{ __html: content }}></div>
```

Comment:
As its name says, `dangerouslySetInnerHTML` should be avoided 😅 Another options is to use an iframe and its `srcdoc` attribute (see: https://www.w3schools.com/tags/att_iframe_srcdoc.asp)

```
<iframe srcdoc={ content }></iframe>
```